runtime/compiler/env/j9method.cpp

```diff
@@ -3957,6 +3958,18 @@ void TR_ResolvedJ9Method::construct()
       {TR::unknownMethod}
       };
 
+   static X VirtualThreadMethods [] =
+      {
+      {  TR::java_lang_VirtualThread_runWith,       7,  "runWith",         (int16_t)-1, "*"},
```

Comment:
Just curious about the whitespace here.  I imagine it is for alignment, but since these are the only entries in these tables it is hard to see what they are being aligned with.

Reply:
Fixed.